CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Git LFS Changelog
 
+## 3.5.0 (28 February 2024)
+
+This release is a feature relerase which includes support for LoongArch and
+RISC-V Linux binary tarballs, `FETCH_HEAD` as a remote source (from a plain
+`git fetch`), better support for detection of the system gitattributes file,
+and configuration options for the SSH protocol.  In this release, the
+`FETCH_HEAD` support is experimental and subject to change.
+
+We would like to extend a special thanks to the following open-source
+contributors:
+
+* @jochenhz for improvements to `git lfs prune`
+* @murez for improvements to our installation script
+* @qiangxuhui for tooling to build 64-bit LoongArch packages
+* @AaronDewes for tooling to build 64-bit RISC-V packages
+* @b-camacho for allowing `FETCH_HEAD` as a fallback remote source
+* @tigsikram for fixing some broken links
+* @aymanbagabas for fixing our SSH protocol documentation
+* @KyleFromKitware for improvements to the pure SSH protocol
+* @Juneezee for several code cleanups
+* @cmaves for improving performance of our progress indications
+* @QuLogic for improving completions and updating docs
+
+### Features
+
+* Add --verify-unreachable option to LFS prune #5648 (@jochenhz)
+* attribute: warn if config exists after uninstalling #5635 (@bk2204)
+* Initialize sessions lazily #5634 (@bk2204)
+* Add a `--local` argument to install.sh #5618 (@murez)
+* Provide loong64 Binary Support #5607 (@qiangxuhui)
+* Improve locking performance #5561 (@bk2204)
+* Allow configuring the SSH protocol #5555 (@bk2204)
+* Add `FETCH_HEAD` as fallback remote source #5357 (@b-camacho)
+* Use `git var` to find system gitattributes file #5412 (@bk2204)
+* Add RISC-V support #5438 (@AaronDewes)
+
+### Bugs
+
+* Add support for homedir expansion in SSL key and cert paths #5657 (@bk2204)
+* Display correct status information when `git lfs ls-files` run in subdirectory #5653 (@chrisd8088)
+* Fix git lfs prune is deleting staged files in the index #5637 (@jochenhz)
+* Report invalid ref in `git lfs push` command #5639 (@chrisd8088)
```

Comment:
This PR built on one we didn't merge but which was authored by a new contributor, who also reported the bug, so it might be nice to include them in these notes somehow.

Reply:
I've included contribution credits for this contributor and the one below.  Thanks for reminding me about this.

Reply:
Awesome, thank you so much!